marked-element.html

```diff
@@ -118,6 +118,19 @@
         observer: 'render',
         type: Boolean,
         value: false
+      },
+      /**
+       * Callback function invoked by Marked after HTML has been rendered.
+       * It must take two arguments: err and text and must return the resulting text.
+       */
+      callback: {
+        observer: 'render',
+        type: Function,
+        value: function() {
+          return function(err, text) {
+            return text;
```

Comment:
Do we want errors to be swallowed by default? I assume this changes marked's behavior to propagate errors asynchronously instead of throwing them synchronously. Can this be null/undefined instead?